.gitignore

```diff
@@ -187,6 +187,9 @@ node_modules/
 /ajax/libs/react-data-grid/*-dm*/
 /ajax/libs/jquery-easing/1.3.0/
 /ajax/libs/jquery-easing/1.3.1/
+/ajax/libs/teadao/0.1.0/
+/ajax/libs/teadao/0.4.0/
+/ajax/libs/teadao/0.5.0/
```

Comment:
I think we don't also need to ignore `v0.1.0` because there is no `teadao*` under `library`. It'll fetch nothing in that version, too. Is it correct?